### PR TITLE
chore: weekly image cve patching from Sec Pipelines

### PR DESCRIPTION
**Pull request by HDYA**
weekly image cve patching from Sec Pipelines

### CODE REVIEW

File: vhdbuilder/packer/kube-proxy-images.json [hunk 1]
```diff
@@ -6,17 +6,20 @@
         "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:v*",
         "amd64OnlyVersions": [],
         "multiArchVersions": [
+          "1.24.10-hotfix.20230509-2",
           "1.24.10-hotfix.20230509.1",
+          "1.24.9-hotfix.20230509-2",
           "1.24.9-hotfix.20230509.1",
+          "1.25.5-hotfix.20230509-3",
           "1.25.5-hotfix.20230509.1",
-          "1.25.5-hotfix.20230509.2",
```
Comment (HDYA) [inline vhdbuilder/packer/kube-proxy-images.json:12]: note: as reported by trivy, there is no different in terms of CVE between `1.25.5-hotfix.20230509.1` and `1.25.5-hotfix.20230509.2`, let's skip it and jump to `1.25.5-hotfix.20230509.3`

**Review by andyzhangx — APPROVED** [review]

File: vhdbuilder/packer/components.json [hunk 2]
```diff
@@ -67,6 +67,7 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
+        "v1.3.3-2",
```
Comment (andyzhangx) [inline vhdbuilder/packer/components.json:70]: why it's not `v1.3.3.2`, is `v1.3.3-2` identical to `v1.3.3.2`?

Comment (andyzhangx) [reply vhdbuilder/packer/components.json:70]: btw, I am going to update v1.3.4 release this week: https://github.com/kubernetes-sigs/secrets-store-csi-driver/releases/tag/v1.3.4, will overwrite `v1.3.3-2` directly

Comment (andyzhangx) [reply vhdbuilder/packer/components.json:70]: here is the PR: https://github.com/Azure/AgentBaker/pull/3315

Comment (HDYA) [reply vhdbuilder/packer/components.json:70]: * yes, they're identical, in the future we'll be using v1.3.3-2. "v1.3.3.1 is not compatible with semantic versioning libraries used by many tools (specifically Helm), whereas v1.3.3-1 is. ", according to Jeremy regarding this change.
* okay, will update corresponding PR too

**Pull request merged**